cloud/deploy/deploy.go

```diff
@@ -676,25 +680,54 @@ func buildImage(path, currentVersion, deployImage, imageName string, dagDeployEn
 	return version, nil
 }
 
-// Deploy the image
-func imageDeploy(imageCreateResID, deploymentID, repository, nextTag, description string, dagDeployEnabled bool, client astro.Client) error {
-	imageDeployInput := astro.DeployImageInput{
-		ImageID:          imageCreateResID,
-		DeploymentID:     deploymentID,
-		Repository:       repository,
-		Tag:              nextTag,
-		DagDeployEnabled: dagDeployEnabled,
-		Description:      description,
-	}
-	resp, err := client.DeployImage(&imageDeployInput)
+// update deploy
+func updateDeploy(deployID, deploymentID, organizationID, dagTarballVersion string, dagDeploy bool, coreClient astrocore.CoreClient) error {
+	UpdateDeployRequest := astrocore.UpdateDeployRequest{}
+	if dagDeploy {
+		UpdateDeployRequest.DagTarballVersion = &dagTarballVersion
+	}
+	resp, err := coreClient.UpdateDeployWithResponse(httpContext.Background(), organizationID, deploymentID, deployID, UpdateDeployRequest)
 	if err != nil {
 		return err
 	}
-
-	fmt.Println("Deployed Image Tag: ", resp.Tag)
+	err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+	if err != nil {
+		return err
+	}
+	if resp.JSON200.DagTarballVersion != nil {
+		fmt.Println("Deployed DAG bundle: ", &resp.JSON200.DagTarballVersion)
```

Comment:
This is not printing the tarball version - `Deployed DAG bundle:  0x14000224560`